mkdocs.yml

```diff
@@ -6,7 +6,7 @@ theme:
    #  custom_dir: 'theme'
 
 nav:
-  - Home: 'https://www.freeflarum.com/docs/'
+  - Home: 'index.md'
```

Comment:
I updated the homepage link in the navbar to support local development of the docs, but the problem is it might broke it when deploying to the FreeFlarum website unless it's on `docs.freeflarum.com`.

Reply:
@CWKevo Because of this change, should we move the docs to `docs.freeflarum.com`?

I'm not yet tested about the consequences of this.

Reply:
I don't think that that's necessary either, but you better ask @gwillem about this.